chia/server/address_manager_store.py

```diff
@@ -10,10 +8,54 @@
     AddressManager,
     ExtendedPeerInfo,
 )
+from chia.util.ints import uint64
+from chia.util.path import mkdir
+from chia.util.streamable import streamable, Streamable
+from chia.util.write_file import write_file
+from dataclasses import dataclass
+from pathlib import Path
+from timeit import default_timer as timer
+from typing import Any, Dict, List, Optional, Tuple
 
 log = logging.getLogger(__name__)
 
 
+@dataclass(frozen=True)
+@streamable
+class PeerDataSerialization(Streamable):
+    """
+    Serializable property bag for the peer data that was previously stored in sqlite.
+    """
+
+    metadata: List[Tuple[str, str]]
+    nodes: List[Tuple[uint64, str]]
+    new_table: List[Tuple[uint64, uint64]]
+
+
+async def makePeerDataSerialization(
+    metadata: List[Tuple[str, Any]], nodes: List[Tuple[int, ExtendedPeerInfo]], new_table: List[Tuple[int, int]]
+) -> PeerDataSerialization:
```

Comment:
is there a reason not to just return `bytes` here directly? as far as I can tell, in all places we call this function we immediately turn the `PeerDataSerialization` object into `bytes`

Reply:
Sounds reasonable